streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/FunctionConversions.scala

```diff
@@ -32,7 +32,13 @@ import java.lang.{Iterable => JIterable}
  * For Scala 2.11, most of these conversions need to be invoked explicitly, as Scala 2.11 does not
  * have full support for SAM types.
  */
-object FunctionConversions {
+private[scala] object FunctionConversions {
```

Comment:
Now it's not public anymore

Reply:
Ah! I misread this before. This class was already defaulted to public visibility, and we can't retract it now, without a KIP.

I recommend moving the `private[streams]` to just the new method.

Sorry for the confusion.

( I wish I had noticed before that the whole object was public unnecessarily )